### ISSUE TITLE
Don't allow block Transforms from Query Loop Item

### ISSUE THREAD

**Issue by diggeddy** [tomusborne/generateblocks#1443]
### Description
<img width="380" alt="Screenshot 2024-11-06 at 01 26 59" src="https://github.com/user-attachments/assets/72e8fed4-095e-4d9a-99e6-231e6d3e8967">

Reduce risk of breakage by removing the block transforms from the Query Loop Item block

**Comment by tomusborne** [comment]
I don't believe it's possible to disable these core transforms, unfortunately.
